manifest.xml

```diff
@@ -1,20 +1,28 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.0" encoding="UTF-8"?>
 <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
-	<name>Community Theme</name>
-	<author>Core developers</author>
+	<name>Star Admin Theme</name>
+	<id>star-admin-theme</id>
```

Comment:
this id doesn't match the repo name or the `composer.json` name which will cause installation issues